src/coreclr/src/tools/Common/TypeSystem/Common/InstantiatedType.cs

```diff
@@ -146,9 +146,9 @@ public override IEnumerable<MethodDesc> GetMethods()
         }
 
         // TODO: Substitutions, generics, modopts, ...
```

Comment:
What todos are left? Just modopts?

Reply:
Actually, modopts work now. The only thing left is complex multidimensional array shape.